spack/repo/packages/py-pyprecice/package.py

```diff
@@ -32,6 +33,8 @@ class PyPyprecice(PythonPackage):
     version("2.0.0.1", sha256="96eafdf421ec61ad6fcf0ab1d3cf210831a815272984c470b2aea57d4d0c9e0e")
 
     depends_on("precice@develop", when="@develop")
+    depends_on("precice")
```

Comment:
Why do we need this `depends_on("precice")` statement? I do not find this statement in the [upstream recipe](https://github.com/spack/spack/blob/develop/var/spack/repos/builtin/packages/py-pyprecice/package.py).

Reply:
Oops!